Jenkinsfile

```diff
@@ -304,6 +304,6 @@ def assertCustomErrorPage() {
 
 def followDefaultPageRedirect(base, redirectPage) {
   def url = redirectPage.split('<meta http-equiv=\"refresh\" content=\"0; URL=')[1]
-  url = base + url.split('\" />')[0]
+  url = base + url.split('\"/>')[0]
```

Comment:
i think it should be `'\">'`

Reply:
Du hast recht. Ist das valides HTML, dass du generierst?

Reply:
html `meta` tag is a void element so it does not have (need) a closing tag. idk why we had it before

Reply:
No longer XML conform - since I don't see a use case for this, it's ok for me.